src/subprocess/runner.ts

```diff
@@ -2,6 +2,7 @@ import * as CircularJSON from 'circular-json';
 import * as Mocha from 'mocha';
 import { IRunner, reporters } from 'mocha';
 import * as yargs from 'yargs';
+import applyGrepPattern from '../bin/options/grep';
```

Comment:
If you don't mind could you please put this method (applyGrepPattern) into `../utils` as we do it with other methods common for the main and subprocesses? https://github.com/qatechniker/mocha-parallel-tests/blob/459c70c4f1f8d7a8cbba3870d6f5f2706ee7e0d5/src/subprocess/runner.ts#L19